subsys/bluetooth/controller/ll_sw/ull_df.c

```diff
@@ -802,17 +800,17 @@ static bool pdu_ext_adv_is_empty_without_cte(const struct pdu_adv *pdu)
  * advertising chain. If particular PDU is empty (holds cte_info only) it will be removed from
  * chain.
  *
- * @param lll_sync Pointer to periodic advertising sync object.
- * @param pdu_prev Pointer to a PDU that is already in use by LLL or was updated with new PDU
- *                 payload.
- * @param pdu      Pointer to a new head of periodic advertising chain. The pointer may have
- *                 the same value as @p pdu_prev, if payload of PDU pointerd by @p pdu_prev was
- *                 already updated.
+ * @param lll_sync      Pointer to periodic advertising sync object.
+ * @param[out] pdu_prev Pointer to a PDU that is already in use by LLL or was updated with new PDU
+ *                      payload. After return points to last PDU in a previous chain.
+ * @param[out] pdu      Pointer to a new head of periodic advertising chain. The pointer may have
+ *                      the same value as @p pdu_prev, if payload of PDU pointerd by @p pdu_prev was
+ *                      already updated. After return points to last PDU in a new chain.
  *
  * @return Zero in case of success, other value in case of failure.
  */
 static uint8_t rem_cte_info_from_per_adv_chain(struct lll_adv_sync *lll_sync,
-					       struct pdu_adv *pdu_prev, struct pdu_adv *pdu)
+					       struct pdu_adv **pdu_prev, struct pdu_adv **pdu)
```

Comment:
The setting of both `pdu_prev` and `pdu` are done in the loop. Is there any case that either of these would not be set? Should we set them to NULL before the loop starts?

Reply:
They are actually in/out params. I have to update the header function description. 
They are already set before the function is called.

Reply:
applied suggestion

Reply:
Ah, so they are simply overwritten in this function then? 

Good idea to use something like `[in-out]` instead of just `[out]` :)